synology-packages.md

```diff
@@ -51,26 +50,35 @@ Bubblewrap allows us to run programs in a basic container so that we can use new
 
 ## Bubblewrap Manual Step - Recommended Option
 
-1. Create a scheduled task within DSM:
+1. Create a triggered task within DSM:
 	- User: `root`
   	- Event: `Boot-up`
-1. For the `Run Command` enter:
+2. For the `Run Command` enter:
 
 ```bash
 chown root:root /volume1/@appstore/bubblewrap/bin/bwrap
 chmod u+s /volume1/@appstore/bubblewrap/bin/bwrap
 ```
 
+3. Save triggered task, and run it once.
+
+![triggered_task.png](/assets/synology/triggered_task.png)
+
 ![create_task1.png](/assets/synology/create_task1.png)
 
 ![create_task2.png](/assets/synology/create_task2.png)
 
+![run_task.png](/assets/synology/run_task.png)
+
 ## Bubblewrap Manual Step - Advanced Option
 
 1. [Login to your Synology via SSH and elevate to `root`](https://kb.synology.com/en-global/DSM/tutorial/How_to_login_to_DSM_with_root_permission_via_SSH_Telnet)
-1. Execute 
+2. Execute following commands:
+
+```bash
+sudo chown root:root /volume1/@appstore/bubblewrap/bin/bwrap
```

Comment:
Why split? Just leave this as a single block

Reply:
I split is, because it doesn't always work correctly when combined.
You need `sudo` to run the command, and it might fail to run the second command when it requests for the root password.